integration/benchmark/follower_test.go

```diff
@@ -83,12 +86,12 @@ func TestNopTxFollower(t *testing.T) {
 		WithInteval(1*time.Hour),
 	)
 	require.NoError(t, err)
-	unittest.AssertClosesBefore(t, f.Follow(flowsdk.Identifier{}), 1*time.Second)
+	unittest.AssertReturnsBefore(t, func() { <-f.Follow(flowsdk.Identifier{}) }, 1*time.Second)
 
 	// test that multiple Stops are safe
 	f.Stop()
 	f.Stop()
 
 	// test that all further Follow calls return immediately
-	unittest.AssertClosesBefore(t, f.Follow(flowsdk.Identifier{}), 1*time.Second)
+	unittest.AssertReturnsBefore(t, func() { <-f.Follow(flowsdk.Identifier{}) }, 1*time.Second)
```

Comment:
shouldn't the timeout be much less than 1 second since it's expected to return right away?

Reply:
here it does not matter since channel is closed immediately.  Sleep here is only needed for the race in `AssertReturnsBefore`.